safe/definitions/__init__.py

```diff
@@ -15,6 +15,7 @@
 from safe.definitions.exposure import *
 from safe.definitions.exposure_classifications import *
 from safe.definitions.fields import *
+from safe.definitions.field_groups import *
```

Comment:
__[flake8]__
* [F403] 'from safe.definitions.field_groups import *' used; unable to detect undefined names
* [F401] 'safe.definitions.field_groups.*' imported but unused

<!-- Comment by [SideCI](https://sideci.com) -->